app/assets/stylesheets/helpers/_print-base.scss

```diff
@@ -101,8 +101,6 @@ table {
 #global-breadcrumb,
 #global-browser-prompt,
 #user-satisfaction-survey,
-.print-link,
-.report-a-problem-toggle,
-.report-a-problem-container {
+.print-link {
```

Comment:
Aside: I've noticed we aren't hiding the new question and forms in the print view